src/main/scala/mesosphere/marathon/DebugConf.scala

```diff
@@ -53,20 +53,20 @@ trait DebugConf extends ScallopConf {
 class DebugModule(conf: DebugConf) extends AbstractModule {
   override def configure(): Unit = {
     //set trace log levelN
-    conf.logLevel.get.foreach { levelName =>
+    conf.logLevel.toOption.foreach { levelName =>
       val level = Level.toLevel(if ("fatal".equalsIgnoreCase(levelName)) "fatal" else levelName)
       LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME) match {
         case l: ch.qos.logback.classic.Logger => l.setLevel(level)
         case _ => println(s"WARNING: Could not set log level to $level!")
       }
     }
 
-    conf.logstash.get.foreach {
+    conf.logstash.toOption.foreach {
```

Comment:
I think you can do `conf.logstash.foreach { ... }`. At least the [docs](http://scallop.github.io/scallop/org/rogach/scallop/ScallopOption.html#foreach(f:A=%3EUnit):Unit) suggest that.